### PR TITLE
Added a temporary field used by `Receipt::Panic` to fill `contract_id` if any

### PR DESCRIPTION
**Pull request by xgreenx**
Fix for https://github.com/FuelLabs/fuel-vm/pull/255 and https://github.com/FuelLabs/fuel-vm/pull/233

### CODE REVIEW

File: src/util.rs [hunk 1]
```diff
@@ -409,14 +409,24 @@ pub mod test_helpers {
...
+        {
             assert_eq!(
                 &expected_reason,
                 reason.reason(),
                 "Expected {}, found {}",
                 expected_reason,
                 reason.reason()
             );
+            match expected_reason {
+                PanicReason::ContractNotInInputs => assert!(contract_id.is_some()),
```
Comment (Voxelot) [inline src/util.rs:427]: Maybe we should also have tests that the contract id is set to an non-zero value?

**Review by Salka1988 — COMMENTED** [review]
LGTM!

File: src/util.rs (hunk 1 above)
Comment (xgreenx) [reply src/util.rs]: Checked `contract_id` for `LCD` opt code

**Review by Voxelot — APPROVED** [review]
